src/devtools/useAtomDevtools.ts

```diff
@@ -2,46 +2,15 @@ import { useEffect, useRef } from 'react'
 import { useAtom } from 'jotai'
 import type { Atom, WritableAtom } from 'jotai'
 import type { Scope, SetAtom } from '../core/atom'
-
-type Config = {
-  instanceID?: number
-  name?: string
-  serialize?: boolean
-  actionCreators?: any
-  latency?: number
-  predicate?: any
-  autoPause?: boolean
-}
-
-type Message = {
-  type: string
-  payload?: any
-  state?: any
-}
-
-type ConnectionResult = {
-  subscribe: (dispatch: any) => () => void
-  unsubscribe: () => void
-  send: (action: string, state: any) => void
-  init: (state: any) => void
-  error: (payload: any) => void
-}
-
-type Extension = {
-  connect: (options?: Config) => ConnectionResult
-}
+import { ConnectResponse, Message } from './types'
 
 export function useAtomDevtools<Value, Result extends void | Promise<void>>(
   anAtom: WritableAtom<Value, Value, Result> | Atom<Value>,
   name?: string,
   scope?: Scope
 ): void {
-  let extension: Extension | undefined
-  try {
-    extension = (window as any).__REDUX_DEVTOOLS_EXTENSION__ as Extension
-  } catch {
-    // ignored
-  }
+  let extension = window?.__REDUX_DEVTOOLS_EXTENSION__
```

Comment:
Do we need `?` optional chaining?
Also, please keep try-catch, which is important.